### DISCUSSION TITLE
Option to sort pull requests by date merged

### DISCUSSION THREAD

**Discussion by RykaC** [community/community#30073] [Pull Requests]
There should be an option to sort the pull requests by the date that they were merged. Right now you can only sort by date created
<img width="437" alt="Screenshot 2022-08-19 at 12 55 14" src="https://user-images.githubusercontent.com/78734752/185613670-6c2735f0-e6de-40c0-a7ef-8ebea5225bf4.png">

**Comment by queenofcorgis** [comment]
Hey there! Thanks for the feedback, I've labelled this Discussion so it can go to the proper team(s).

**Comment by kylewhitaker** [comment]
Long time user, first time caller. I came here to make this exact same feature request.

**Comment by delanoyoder** [comment]
This feature seems like a no-brainer and a powerful tool for debugging.
